apps/daimo-mobile/src/view/shared/Badge.tsx

```diff
@@ -16,8 +13,10 @@ export function Badge({
   color?: string;
   bgColor?: string;
```

Comment:
might be worth replacing `color/bgColor` argument with a semantic color name > look up in theme

Reply:
actually, fine as is. i think bgColor argument can be deleted, it's never passed